README.md

```diff
@@ -1,2 +1,210 @@
 # Math-Magicians
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Thank you for the advice, and thank you for the approval 😄.